Libraries/VtkVgCore/vtkVgTypeRegistry.h

```diff
@@ -34,6 +34,10 @@ class vtkVgTypeRegistry : public vtkObject, vtkVgTypeRegistryBase
   void SetType(int index, const T& type);
   virtual void SetEntityType(int index, const vgEntityType& type);
 
+  // Description:
+  // Get index of registered track type or -1 if not found.
+  int GetTypeIndex(const char* id) const;
```

Comment:
nit, but `id` here and `name` in definition

Reply:
Oh, I see what happened; `vgTrackType` overloads `GetId()` to return the name (literally, the body of the method is `return this->GetName()`), while `vgEventType` has `int GetId()`. The definition, accordingly, had to switch to `name`/`GetName()`, but the declaration (i.e. this) got moved straight from `vtkVgTrackTypeRegistry` and missed the change.

Nice catch! Will fix it...